enhancements/authentication/login-logout-events.md

```diff
@@ -102,6 +102,92 @@ const (
 )
 ```
 
+We propose to add additional information that helps to identify login and login failures to the audit log.
+
+In case that the authentication happens through the `oauth-server`, we suggest to add:
+
+- `authentication.openshift.io/username`, which is the username for the authentication attempt.
+- `authentication.openshift.io/authn-method`, which mentions the OAuth2 authentication method in use.
```

Comment:
This can be implied based on the request URL, no?
Why is this important?
Should it be `oauth-method` instead of `authn-method`, then?

Reply:
Yeah, this is really a thing that makes me wonder:

- UX for "just reading the annotations" during a "wake up call in the night"
vs
- redundant information that could (Murphy's Law) contradict itself.

The method and the decision can be deduced from the other values.

Reply:
I didn't want say "oauth-method" as I would rather think in terms of implicit grant or code grant when thinking of oauth-methods. So as we pick "just" the authentication method, I went for authn.

Reply:
I think that for now we can just go with username and decision to keep things minimal. We can add more later if there's such a request